yoti-sdk-api/src/main/java/com/yoti/api/client/AgeVerification.java

```diff
@@ -0,0 +1,36 @@
+package com.yoti.api.client;
+
+/**
+ * Wraps an 'Age Verify/Condition' attribute to provide behaviour specific to verifying someone's age
+ */
+public interface AgeVerification {
+
+    /**
+     * The type of age check performed, as specified on dashboard. Currently this might be 'age_over' or 'age_under'
+     *
+     * @return The type of age check performed
+     */
+    String getCheckPerformed();
```

Comment:
I don't really like the naming.
I'm more for something like `getCheckType()`.
I know you don't want to use enums, but I would still change the name and make it return a string that is a constant.